### PR TITLE
[django] Added support for Django 2.2 #93

### LINKED ISSUE #93: [django] Add support for django 2.2
Add support for django 2.2.

### PR DESCRIPTION
**Pull request by oltarzewskik**
Added support to `Django 2.2` and closes #93

### CODE REVIEW

**Comment by oltarzewskik** [comment]
@nemesisdesign Django 2.2 no longer supports `sqlite` verision 3.8.2 (it is the reason why build is failling). And I found out that it is not pretty easy to change version of `sqlite` if one is deprecated (https://askubuntu.com/a/580400) so what we should do in that case? Should we only write in `README` that supported versions of `sqlite` is `>=3.8.2` or do something more complex?

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-django>=1.11,<2.2
+django>=1.11,<3.0
```
Comment (nemesisdesign) [inline requirements.txt]: don't use `3.0`, use `2.3`, same applies for the other occurrences

**Comment by oltarzewskik** [comment]
@nemesisdesign I handled problem of version of `sqlite` by changing version of ubuntu used by travis to 16.04 which have more recent version of `sqlite` if there is any specific reason of using ubuntu 14.04 I think that I can handle it too, by using this guy's [solution](https://github.com/cmu-is-projects/ferry/pull/72/files)

And referring to using `2.3` instead of `3.0`, I am not sure that there is any sense of using this since according to wikipedia (https://en.wikipedia.org/wiki/Django_(web_framework)#Version_history) the version of `django` after `2.2` will be `3.0` not `2.3`

File: requirements.txt (hunk 1 above)
Comment (nemesisdesign) [reply requirements.txt]: @oltarzewskik :point_up:

Comment (oltarzewskik) [reply requirements.txt]: @nemesisdesign Ok, changed

**Pull request merged**